tests/stats_test.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import pandas as pd
+from base_test import ArkoudaTest
+from context import arkouda as ak
+
+
+class StatsTest(ArkoudaTest):
+    def setUp(self):
+        ArkoudaTest.setUp(self)
+        self.x = ak.arange(10, 20)
+        self.pdx = pd.Series(np.arange(10, 20))
+        self.y = ak.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48])
+        self.pdy = pd.Series(np.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48]))
+        self.u = ak.cast(self.y, ak.uint64)
+        self.pdu = pd.Series(np.array([2, 1, 4, 5, 8, 12, 18, 25, 96, 48], dtype=np.uint64))
+        self.b = ak.arange(10) % 2 == 0
+        self.pdb = pd.Series(np.arange(10) % 2 == 0)
+        self.f = ak.array([i**2 for i in range(10)], dtype=ak.float64)
+        self.pdf = pd.Series(np.array([i**2 for i in range(10)], dtype=np.float64))
+        self.s = ak.array([f"string {i}" for i in range(10)])
+        self.c = ak.Categorical(self.s)
+
+    def test_mean(self):
+        self.assertEqual(self.x.mean(), self.pdx.mean())
+        self.assertEqual(self.y.mean(), self.pdy.mean())
+        self.assertEqual(self.u.mean(), self.pdu.mean())
+        self.assertEqual(self.b.mean(), self.pdb.mean())
+        self.assertEqual(self.f.mean(), self.pdf.mean())
+
+    def test_var(self):
```

Comment:
Should this also test against numpy? It looks like we are only testing pandas.

Reply:
done!